CRM/Upgrade/Incremental/sql/5.23.alpha1.mysql.tpl

```diff
@@ -9,3 +9,5 @@ ALTER TABLE civicrm_payment_processor ALTER COLUMN is_active SET DEFAULT 1;
 ALTER TABLE civicrm_payment_processor ALTER COLUMN is_test SET DEFAULT 0;
 ALTER TABLE civicrm_payment_processor_type ALTER COLUMN is_active SET DEFAULT 1;
 ALTER TABLE civicrm_payment_processor_type ALTER COLUMN is_default SET DEFAULT 0;
+
+ALTER TABLE civicrm_note MODIFY COLUMN modified_date TIMESTAMP COMMENT 'When was this note last modified/edited';
```

Comment:
don't we need to set the default as current_timestamp()?

Reply:
@seamuslee001 MariaDB seems to do that by default .. I've added the default to the PR